tests/fixtures/oneroster_client.php

```diff
@@ -37,4 +38,5 @@
  */
 abstract class oneroster_client implements client_interface, rostering_client_interface {
     use root_oneroster_client;
+    use csv_oneroster_client;
```

Comment:
Maybe this should be its own fixture so there aren't side effects on the Oauth classes tests